azure_costs_exporter/views.py

```diff
@@ -11,14 +12,19 @@ def health():
 @bp.route("/metrics", methods=['GET'])
```

Comment:
Given the goal to run this using Gunicorn and with the addition of a `/health` endpoint, I think it is OK to just use `flask` here. 

An other alternative would be to use the existing `make_wsgi_app` call (https://github.com/prometheus/client_python#wsgi). However I have never tested this and it would still require custom code to add /health.